### ISSUE TITLE
Relation for devices and the materials that the devices uses

### ISSUE THREAD

**Issue by wdduncan** [oborel/obo-relations#497]
Is there a relation that holds between a device, such as a hematology analyzer, and the specimen/sample that device uses to produce the result?  
Prov has a [used](https://www.w3.org/TR/prov-o/#used) predicate, but that holds between an activity and the thing the activity used.

Could add a `utilizes` relation? E.g, `hemotology-device#1` `utilizes` `blood-sample#1`

cc @cmungall @matentzn @balhoff @dosumis

**Comment by wdduncan** [comment]
P.S. I realize that this can be done with a complex pattern like

```
hemotology-device#1 has-disposition 
  [a disposition; realized in [a analysis-process; has-input blood-sample#1]]
```
But that seems like quite a lot to have to say.

**Comment by linikujp** [comment]
I think the device utilizes specimen is a shortcut relation. This is my use case:
a diagnostic testing device - is capable of - RT-PCR assay; -has specified input - a specimen

**Comment by cmungall** [comment]
I think the property chain makes sense, but the name 'utilizes' seems more general

how about naming this "capable-of-has-specified-input"?

Note you won't be able to make this in RO without importing OBI which introduces modularity issues

what about capable-of-has-input?

**Comment by linikujp** [comment]
capable-of-has-input is not human or non-ontologist understandable.  I
would prefer other more user-friendly labels.
Regarding applying this as a RO term, we may need to think about it. It
seems like a diagnostic testing device specific term.

**Comment by wdduncan** [comment]
I agree with @linikujp. "capable-of-has-input" will not be an easily accessible term. We can, of course, define a property chain on the relation:
```
'capable of' o 'has input'
```
I'm fine with with label `utilizes`. Another option might be `processed by`.

**Comment by linikujp** [comment]
This situation may change. Some diagnostic testings do not have the
capability of testing. The kit is to collect samples, and then sent to a
laboratory where a sequencing machine or a RT-PCR device to run the RT-PCR
test... Some home-use kit may have the capacity of running kinds of RT-PCR
testing or antigen-antibody detecting tests...
The solution of  'capable of' o 'has input' will work on testings that have
a component of assays.
Other diagnostic testing kits have to combine with certain off-site (of
collection) equipment in a controlled lab environment.

Thanks,
Asiyah
.

**Comment by wdduncan** [comment]
The `'capable of' o 'has input'` property chain does not need to be restricted to assays. It is restricted to cases in which a material entity is an input to a process.

The collection of specimens is an interesting case. There is come kind of [specimen collection process](http://www.ontobee.org/ontology/OBI?iri=http://purl.obolibrary.org/obo/OBI_0000659) going on. Not sure if you want to say that the device `utilizes` specimen, though.

**Comment by linikujp** [comment]
Yes, aware of the specimen collection process. The device *utilizes *specimen
is important to capture in the diagnostic testing modeling use case.

**Comment by wdduncan** [comment]
I think for cases in which there are outputs that are produced by the device, `utilizes` seems fine. For cases of storing samples, `utilizes` seems wrong. For example, a freezer `utilizes` freon to produce/output a cold environment. It seems to odd to say that the freezer `utilizes` the items stored inside of it. `Located in` or some relation like it, seems more appropriate.

**Comment by linikujp** [comment]
Bill, good thoughts!
The whole diagnostic testing may need to include the specific RT-PCR
machine as part of it. What machines should be paired with each diagnostic
testing kit are specified in the EUA documents. So the *'utilizes'* may
still work here.
Plus, the collection kit includes sample collection, sample transportation,
and sample storage if it is not a self-testing kit.
Therefore,

**Comment by linikujp** [comment]
OBI call today (01/10/2022) discovered the conflict:
OBI has been using "utilizes" linking process to device long time ago, and we are using utilizes for device to material. The OBI accidentally obsoleted the "utilizes" relation and they are reviving this relation.
Both are short relations.  @wdduncan we need to resolve the conflict. We may create a new relation specifically for device to material rather than fix the utilizes. And I could propose to OBI that utilizes will be the reverse relation of 'capable of' in our setting.

**Comment by linikujp** [comment]
https://github.com/obi-ontology/obi/issues/1480

**Comment by linikujp** [comment]
@wdduncan  Can we finalize the relation from device to specimen?

**Comment by wdduncan** [comment]
@linikujp Sorry, I am not following you. Are you suggesting a new label?

**Comment by linikujp** [comment]
utilizes is in conflict with OBI use. Please see previous discussion. Yes to your question. Shall we use another label?

**Comment by wdduncan** [comment]
Yes. I am aware of the other discussion. Right now, utilizes is still obsolete in OBI. So, there isn't a conflict. I made a number of label suggestions. What label do you prefer?

**Comment by linikujp** [comment]
I like "device utilizes material" this is very straight forward. It was endorsed by Bjoern as well.

**Comment by linikujp** [comment]
@wdduncan can we finalize with creating this relation in RO? After the creation, this ticket can be closed.
Thanks,
Asiyah

**Comment by wdduncan** [comment]
@linikujp Do you have an ORCID?

**Comment by linikujp** [comment]
Yes. It is 0000-0003-2620-0345

**Comment by wdduncan** [comment]
Thanks. I'll use your ORCID as the value in `term editor`. That seems to be preferred now.

**Comment by wdduncan** [comment]
@linikujp See this [PR](https://github.com/oborel/obo-relations/pull/605).

**Comment by linikujp** [comment]
Thank you! That's great.  When will it be ready to use?

**Comment by wdduncan** [comment]
@linikujp Let's wait for the PR be approved.

**Comment by wdduncan** [comment]
@linikujp The relation `device utilizes material` has been added to `ro-edit.owl`. It will be in the next release.
